pkg/sql/catalog/lease/lease.go

```diff
@@ -1726,6 +1738,7 @@ func (m *Manager) refreshLeases(
 					if err := evFunc(desc); err != nil {
 						log.Infof(ctx, "skipping update of %v due to knob: %v",
 							desc, err)
+						continue
```

Comment:
I'm surprised that fixing this omission didn't reveal bugs in existing tests. Oh well.